tests/cpp/test_resize.cpp

```diff
@@ -6040,4 +6084,49 @@ TEST_F(ResizeTest, PropagateResizeThroughMultiplePaths) {
   testValidate(&fusion, outputs.outputs, {t0, t1}, __LINE__, __FILE__);
 }
 
+// Check if vectorization is properly applied even when a resized ID
+// is reachable from vectorized IDs. Pattern extracted from Litgpt
+// LLama RoPE backward.
```

Comment:
nitpick on comment.

This is the case where it's safe to skip the additional resize check. So that means the resized ID is *NOT* reachable from vectorized IDs.